airbyte-integrations/connectors/destination-duckdb/metadata.yaml

```diff
@@ -4,7 +4,7 @@ data:
   connectorSubtype: database
   connectorType: destination
   definitionId: 94bd199c-2ff0-4aa2-b98e-17f0acb72610
-  dockerImageTag: 0.3.0
+  dockerImageTag: 0.4.0
```

Comment:
@elefeint - Since this should be minimally disruptive to users and (I think) doesn't have any major feature changes, wdyt of us using `0.3.1` instead of `0.4.0`? 

I don't feel very strongly either way, so feel free to push back.

Reply:
0.3.1 sounds good; I wondered about this myself

Reply:
done